lib/model.js

```diff
@@ -1090,13 +1090,10 @@ class Model {
     this._dataTypeChanges = {};
     this._dataTypeSanitizers = {};
 
-    this._booleanAttributes = [];
```

Comment:
Keep all these stores, no need to use memoization, you can use `Set`

Reply:
Actually get rid of these changes, better to target them in separate PR. Let's focus on code cleanup in this PR